util/util-functions.js

```diff
@@ -85,4 +85,33 @@ function $(id) {
 	var elem = document.getElementById(id);
 	if(!elem) elem = document.body;
 	return elem;
-}
\ No newline at end of file
+}
+
+function getSalt() {
+	return "VGhlIGZvbGxvd2luZyBkb2Vzbid0IG1lYW4gYW55dGhpbmcuIEl0J3MganVzdCBnYXJiYWdlIGRhdGEgZ2VuZXJhdGVkIGJ5IHlvdXIgY2xpZW50LgoKU25WemRDQmtiMjUwQ2dwVWF6aG5Va1U1VDFaRFJVdERiVVpJVDFST1NsSXhTbmRYYTA1RFRURndWRkZ0TldGWFJrWnVXVlZrVjJWV2NGSlFWREE5";
+	// Random salt. doesnt have a meaning
+}
+
+function encrypt(salt, text) {
+	const textToChars = (text) => text.split("").map((c) => c.charCodeAt(0));
+	const byteHex = (n) => ("0" + Number(n).toString(16)).substr(-2);
+	const applySaltToChar = (code) => textToChars(salt).reduce((a, b) => a ^ b, code);
+
+	return text
+	  .split("")
+	  .map(textToChars)
+	  .map(applySaltToChar)
+	  .map(byteHex)
+	  .join("");
+}
+
+function decrypt(salt, encoded) {
+	const textToChars = (text) => text.split("").map((c) => c.charCodeAt(0));
+	const applySaltToChar = (code) => textToChars(salt).reduce((a, b) => a ^ b, code);
+	return encoded
+	  .match(/.{1,2}/g)
+	  .map((hex) => parseInt(hex, 16))
+	  .map(applySaltToChar)
+	  .map((charCode) => String.fromCharCode(charCode))
+	  .join("");
+}
```

Comment:
Bitte Autor angeben! (Mit Link)
Beispiel:
`Thanks to user1234 on stackoverflow -> https://...`